src/shared/api/validation.ts

```diff
@@ -258,39 +259,75 @@ export function validate(
     if (body) {
         chXHR = Promise.resolve({ body, elapsedTime });
     } else {
-        chXHR = $.ajax({
-            method: 'post',
-            url: url,
-            data: JSON.stringify(params),
-            contentType: 'application/json',
-        }).then((body, state, xhr) => {
-            return { body, elapsedTime: xhr.getResponseHeader('elapsed-time') };
-        });
-    }
-
-    return chXHR
-        .then(({ body, elapsedTime }: any) => {
-            let legacyUrl = url.replace(/column-store\//, '');
-            if (treatmentLegacyUrl[label]) {
-                legacyUrl = treatmentLegacyUrl[label](legacyUrl);
+        if (assertResponse && assertResponse[0] && assertResponse[0].attributeId) {
```

Comment:
not sure i understand the necessity of this code up here.  the clickhouse fetch should be entirely independent of the assertResponse, no?

Reply:
i used it to get the one attribute that is used in assertResponse. so the clinical data counts clickhouse fetch only fetches data for that attribute. then its an easy compare between that response and assertResponse.